assets/js/components/ErrorHandler/ErrorComponent.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+// eslint-disable-next-line @wordpress/dependency-group
+import Button from 'GoogleComponents/button';
+
+export const ThrowError = ( event ) => {
```

Comment:
Nitpicking, but should this not start with a lowercase letter? Shouldn't block this though.

Reply:
Yeah actually, that's totally fair. Next time I make a PR I'll sneak in a case-change 😅